--- MapServer-rel-8-2-2.orig/CMakeLists.txt	2025-01-09 23:39:17.797790900 +0300
+++ MapServer-rel-8-2-2/CMakeLists.txt	2025-01-09 23:43:45.825893500 +0300
@@ -906,6 +906,10 @@
         #4701 : Potentially uninitialized local variable 'name' used (https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4701)
         set_target_properties(mapserver PROPERTIES COMPILE_FLAGS "/EHsc /wd4267 /wd4244 /wd4018 /w14701")
     endif(MSVC)
+    find_package(dlfcn-win32)
+    if(dlfcn-win32_FOUND)
+        ms_link_libraries( dlfcn-win32::dl )
+    endif(dlfcn-win32_FOUND)
 endif (WIN32)
 
 configure_file (
